web-wallet/src/lib/components/Stake/Stake.svelte

```diff
@@ -113,18 +122,74 @@
 	$: fee = formatter(luxToDusk(luxFee));
 	$: maxSpendable = deductLuxFeeFrom(spendable, luxFee);
 	$: minStake = maxSpendable > 0 ? Math.min(defaultMinStake, maxSpendable) : defaultMinStake;
+
+	let hideStakingNoticeNextTime = false;
+
+	function getWizardSteps () {
+		if (flow === "stake") {
+			return hideStakingNotice ? 3 : 4;
+		}
+
+		return 2;
+	}
 </script>
 
 <div class="operation">
-	<Wizard steps={flow === "stake" ? 3 : 2} let:key>
+	<Wizard steps={getWizardSteps()} let:key>
 		{#if flow === "stake"}
+			{#if !hideStakingNotice}
+				<!-- STAKING NOTICE STEP -->
+				<WizardStep
+					step={0}
+					{key}
+					backButton={{
+						action: resetOperation,
+						disabled: false
+					}}
+					nextButton={{
+						action: () => {
+							if (hideStakingNoticeNextTime) {
+								settingsStore.update(store => {
```

Comment:
I have tried to dispatch an event with the updated store data, but it didn't work.

There was a mismatch between the types – the CustomEvent that gets dispatched, that contains the updated store object, and the `store.update(newData)`, which is expecting an `<Updater{newData}> object`.

Maybe we can discuss it additionally. 

@ascartabelli

Reply:
> I have tried to dispatch an event with the updated store data, but it didn't work.
> 
> There was a mismatch between the types – the CustomEvent that gets dispatched, that contains the updated store object, and the `store.update(newData)`, which is expecting an `<Updater{newData}> object`.

I'm not getting the problem. Why you should dispatch the whole store object content?
The event should simply mean that the user clicked "hide staking notice", it's who listens the event that should update the store.
Like we update the operation [here](https://github.com/dusk-network/rusk/blob/master/web-wallet/src/routes/(app)/dashboard/%2Bpage.svelte#L51).